src/main/java/com/mercuryCyclists/Inventory/service/ProductService.java

```diff
@@ -172,4 +177,15 @@ public void deletePart(Long productId, Long partId) {
         if (product == null) return;
         partRepository.deleteById(partId);
     }
+
+    /**
+     * Adds backorder in procurement service
+     * @param sale
+     * @return
+     */
+    public boolean addBackOrder(String sale) {
+        kafkaTemplate.send("backorder", sale);
```

Comment:
from a production standpoint this is so sketchy lol, i still reckon it should be in the sale service as thats where all our validations are. Like a user could just hit the backorder endpoint?

Anyways thats what the lect asked lol, just hoping we dont miss anything